streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -128,9 +128,13 @@
     /** <code>client.id</code> */
     public static final String CLIENT_ID_CONFIG = CommonClientConfigs.CLIENT_ID_CONFIG;
 
+    /** <code>rocksdb.config.setter</code> */
     public static final String ROCKSDB_CONFIG_SETTER_CLASS_CONFIG = "rocksdb.config.setter";
     public static final String ROCKSDB_CONFIG_SETTER_CLASS_DOC = "A Rocks DB config setter class that implements the <code>RocksDBConfigSetter</code> interface";
 
+    /** <code>windowstore.changelog.additional.retention.ms</code> */
+    public static final String WINDOW_STORE_CHANGE_LOG_ADDITIONAL_RETENTION_MS_CONFIG = "windowstore.changelog.additional.retention.ms";
```

Comment:
@dguy Today with zkClient based topic creation, and if we just put whatever users provide in the map via `dataMap.put("config", prop)` in `InternalTopicManager`, upon being notified from the ZK watcher the server will read the configs from the config path, and will ignore any keys that are not recognized. So probably we can just add the guidance in the javadoc string while not enforcing it programmatically.

When we remove the zkClient dependency and use the create topic request (in KIP-4, cc @hjafarpour)